chain/chain/src/chain.rs

```diff
@@ -3451,6 +3451,7 @@ impl<'a> ChainUpdate<'a> {
     where
         F: FnMut(ChallengeBody) -> (),
     {
+        let _span = tracing::debug_span!(target: "runtime", "process_block").entered();
```

Comment:
```suggestion
        let _span = tracing::debug_span!(target: "chain", "process_block").entered();
```

?

Reply:
It feels that this span now duplicates the subsequent log line, so I suggest merging the two. And we probably should be using structured logging capability of tracing to log block.hash, height and such: 

https://docs.rs/tracing/0.1.28/tracing/#recording-fields

(nodex team probably wants some general guidelines with respcet to how we use structured logs)

Reply:
Having general guidelines would certainly be great. And it makes sense for nodeX to own them.
However, the guidelines are rather far out of scope of this PR :)
Trying a few options, I think that including block height in the span is the best tradeoff. It keeps the logs short, but makes them easily parsable.